### PR TITLE
[ Move Example ]Add Steal Or Split Example

### PR DESCRIPTION
**Pull request by WGB5445**

### CODE REVIEW

File: examples/steal_split/Move.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+name = "examples/steal_split"
+version = "0.0.1"
+
+[dependencies]
+MoveStdlib = { git = "https://github.com/rooch-network/rooch.git", subdir = "moveos/moveos-stdlib/move-stdlib", rev = "main" }
+MoveosStdlib = { git = "https://github.com/rooch-network/rooch.git", subdir = "moveos/moveos-stdlib/moveos-stdlib", rev = "main" }
+RoochFramework = { git = "https://github.com/rooch-network/rooch.git", subdir = "crates/rooch-framework", rev = "main" }
+
+[addresses]
+steal_split =  "0x1b1272eece1923fa84926d5c54ed2b3bd00d55713ba41c6d2d398c3cfada4b2"
```
Comment (jolestar) [inline examples/steal_split/Move.toml:11]: Use `steal_split = "_"` for convenient user deployment

**Review by jolestar — COMMENTED** [review]
Need a readme to describe the example

File: examples/steal_split/sources/steal_split.move [hunk 2]
```diff
@@ -0,0 +1,1141 @@
...
+        coin::destroy_freeze_cap(freeze_cap);
+        coin::destroy_mint_cap(mint_cap);
+
+        account_storage::global_move_to(ctx, &signer, State {
+            next_game_id: 0,
+            games: simple_map::create(),
+            cap
+        });
+
+        event::ensure_event_handle<CreateGameEvent>(ctx);
```
Comment (baichuan3) [inline examples/steal_split/sources/steal_split.move:130]: Developers do not need to explicitly call `ensure_event_handle`, this method will be called implicitly by the contract when calling `event::emit`

Comment (WGB5445) [reply examples/steal_split/sources/steal_split.move]: Thanks, I have fixed it.

File: examples/steal_split/Move.toml (hunk 1 above)
Comment (WGB5445) [reply examples/steal_split/Move.toml]: Thanks, It has been updated now.

**Review by jolestar — APPROVED** [review]

**Review by baichuan3 — APPROVED** [review]

**Pull request merged**